packages/common/src/db/queries/Nominator.ts

```diff
@@ -111,7 +111,7 @@ export const setNominatorAvgStake = async (
 export const setTarget = async (
   address: string,
   target: string,
-  now: number,
+  era: number,
```

Comment:
isn't it used with `now`, for example [here](https://github.com/w3f/1k-validators-be/blob/master/packages/core/src/nominator.ts#L431) ?

Reply:
The first dry run place isn't used at all (the dry run stuff can be removed in a future PR, or refactored to be useful in some way). The second is in tests that have been deprecated (and should probably be removed or refactored to work and test properly)